### ISSUE TITLE
ec2 spot instance support

### ISSUE THREAD

**Issue by jmancuso** [GorillaStack/auto-tag#18]
Love the current functionality.... but any plan for spot instance support in the future?

**Comment by em0ney** [comment]
Awesome suggestion.  I added the enhancement label.  I'm going to be reviewing this repository and doing some updates, so I'll try implement this then

**Comment by rayj-pgi** [comment]
I haven't used the spot system so I don't know it very well, but I'm running into two problems. 

1. When I create a spot fleet I see a "RequestSpotFleet" event, but tagging the fleet isn't supported yet according to the docs
1. When the spot fleet launches the spot instance I never see a "RequestSpotInstances" event

**Comment by ElizabethAnthony94** [comment]
Any updates on this?

**Comment by abhianand09** [comment]
Yes that will be very helpful if these events are also handled in the stack
